package.json

```diff
@@ -41,7 +41,7 @@
     "compression": "1.7.3",
     "cors": "2.8.4",
     "deep-extend": "0.6.0",
-    "design-system": "github:Wikia/design-system#15.1.1",
+    "design-system": "github:Wikia/design-system#579db01fe9d8a3e69c5b414d68ec71549e4ddc3a",
```

Comment:
Remember to update after https://github.com/Wikia/design-system/pull/276 is released.